tests/test_devices.py

```diff
@@ -1190,3 +1190,20 @@ async def test_update_stats(unifi_controller: Controller) -> None:
         == "www.microsoft.com"
     )
     assert device.uptime_stats["WAN"].get("monitors")[0].get("type") == "icmp"
+
+
+@pytest.mark.parametrize(("device_payload"), [[GATEWAY_USG3]])
+@pytest.mark.usefixtures("_mock_endpoints")
+async def test_storage(unifi_controller: Controller) -> None:
+    """Test device class storage."""
+    await unifi_controller.devices.update()
+    device = next(iter(unifi_controller.devices.values()))
+
+    assert device.uptime_stats is not None
+    assert len(device.storage) == 2
+
+    assert device.storage[0].get("mount_point") == "/persistent"
```

Comment:
All properties are required so don't use get on them

Reply:
Is it `["property_name"]` as in 56ae1d7e4f9d5b30822e605d499fa85ceb75d01d